pkgs/shells/fish/plugins/fifc.nix

```
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+}:
+buildFishPlugin rec {
+  pname = "fifc";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazorby";
+    repo = "fifc";
+    rev = "v${version}";
+    hash = "sha256-p5E4Mx6j8hcM1bDbeftikyhfHxQ+qPDanuM1wNqGm6E=";
+  };
+
+  meta = with lib; {
+    description = "Fzf powers on top of fish completion engine and allows customizable completion rules";
+    homepage = "https://github.com/gazorby/fifc";
+    license = licenses.mit;
+    maintainers = with maintainers; [haseeb];
```

```suggestion
    maintainers = with maintainers; [ hmajid2301 ];
```
It seems that you have registered yourself as `hmajid2301`.
https://gist.github.com/GrahamcOfBorg/785353fbdab54c9ac52dac6d448821c4#file-output-path-comparison-L70-L75